### ISSUE TITLE
Tutorial16 runtime error

### ISSUE THREAD

**Issue by Yuki-man** [AntonioLonga/PytorchGeometricTutorial#19]
[Tutorial16] Report an error，There is an error in BatchNorm1d part
